src/kernel_abi.h

```diff
@@ -1422,6 +1422,43 @@ struct BaseArch : public wordsize,
     int32_t miimon;
   } ifbond;
   RR_VERIFY_TYPE(ifbond);
+
+  typedef struct timex {
+    int modes;
+    long offset;
```

Comment:
Yes, indeed. Also `modes` should be `unsigned int` though it goes unnoticed since only the size of the structure matters. I'll update the pull request.